src/main/java/org/wikivoyage/listings/validators/LatitudeValidator.java

```diff
@@ -1,14 +1,38 @@
 package org.wikivoyage.listings.validators;
 
+import org.apache.commons.lang.StringUtils;
 import org.wikivoyage.listings.entity.Listing;
 
 public class LatitudeValidator extends SimpleValidator {
+	
+	//0 at the equator; 90 at the respective poles
+	private static final int MIN_LATITUDE = -90;
+	private static final int MAX_LATITUDE = 90;
+	
     @Override
     public void validate(Listing poi) {
-        if (poi.getLatitude() != null && !poi.getLatitude().equals("")) {
+    	String latitude = poi.getLatitude();
+    	if (StringUtils.isEmpty(latitude)) {
+    		//Do nothing; not invalid but no point parsing
+    		return;
+    	}
+    	
+    	//Sanitise the latitude to see if it contained invalid chars. Not trying to fix, just flag as invalid.
+    	// This is more robust than just relying on the NFE because we explicitly know what a latitude SHOULD contain.
+    	String sanitisedLatitude = latitude.replaceAll("[^0-9.-]","");
```

Comment:
Won't the pattern be recompiled on each usage?

Reply:
Declared statically now so it'll only be compiled on instantiation